flink-connectors/flink-connector-kafka/src/test/java/org/apache/flink/connector/kafka/source/enumerator/KafkaEnumeratorTest.java

```diff
@@ -72,14 +73,14 @@ public class KafkaEnumeratorTest {
     private static final boolean INCLUDE_DYNAMIC_TOPIC = true;
     private static final boolean EXCLUDE_DYNAMIC_TOPIC = false;
 
-    @BeforeClass
+    @BeforeAll
     public static void setup() throws Throwable {
```

Comment:
```suggestion
    static void setup() throws Throwable {
```
could be package private